src/docfx/config/ops/OpsAccessor.cs

```diff
@@ -32,12 +32,11 @@ internal class OpsAccessor : ILearnServiceAccessor
 
         public static readonly DocsEnvironment DocsEnvironment = GetDocsEnvironment();
 
-        // TODO: use Azure front door endpoint when it is stable
         private static readonly string s_docsProdServiceEndpoint =
-            Environment.GetEnvironmentVariable("DOCS_PROD_SERVICE_ENDPOINT") ?? "https://op-build-prod.azurewebsites.net";
+            Environment.GetEnvironmentVariable("DOCS_PROD_SERVICE_ENDPOINT") ?? "https://buildapi.docs.microsoft.com";
```

Comment:
Do you still need `DOCS_PROD_SERVICE_ENDPOINT`?

Reply:
This is just a default value, Docs.Build will pass this environment variable, maybe we can still keep this override mechanism?

Reply:
If there is no value in this override, I'd like to remove it for simplicity